pkg/kn/commands/trigger/describe.go

```diff
@@ -105,3 +135,7 @@ func writeTrigger(dw printers.PrefixWriter, trigger *v1beta1.Trigger, printDetai
 		}
 	}
 }
+
+func extractURL(trigger *v1beta1.Trigger) string {
+	return trigger.Status.SubscriberURI.String()
+}
```

Comment:
This is not trigger's URL but the subscriber's, and I dont think we should  expose `-o url` to trigger describe at all.
Discussed this with @kameshsampath the other day on slack, wdyt @rhuss ?

Reply:
I agree, that we should only expose `-o url` to "Addressable" e.g. for objects that have a `status.uri`

Reply:
Got it.Thanks